Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -341,379 +341,501 @@ oanda,USDTRY,forex,USD/TRY,TRY,1,0.00001,1
+gdax,BCHGBP,crypto,Bitcoin Cash-Pound Sterling,GBP,1,0.01,0.01
+gdax,BCHUSD,crypto,Bitcoin Cash-US Dollar,USD,1,0.01,0.01
+gdax,BTCCAD,crypto,Bitcoin-Canadian Dollar,CAD,1,0.00001,0.00000001
+gdax,BTCEUR,crypto,Bitcoin-Euro,EUR,1,0.01,0.001
+gdax,BTCGBP,crypto,Bitcoin-Pound Sterling,GBP,1,0.01,0.001
+gdax,BTCUSD,crypto,Bitcoin-US Dollar,USD,1,0.01,0.001
+gdax,BTCUSDC,crypto,Bitcoin-USDC,USDC,1,0.01,0.001
+gdax,CVCUSDC,crypto,Civic-USDC,USDC,1,0.000001,1.
+gdax,DAIUSDC,crypto,Dai-USDC,USDC,1,0.000001,1.
+gdax,DNTUSDC,crypto,district0x-USDC,USDC,1,0.000001,1.
+gdax,DSHBTC,crypto,Dash-Bitcoin,BTC,1,0.00000001,0.01
+gdax,DSHUSD,crypto,Dash-US Dollar,USD,1,0.001,0.01
+gdax,EOSBTC,crypto,EOS-Bitcoin,BTC,1,0.000001,0.1
+gdax,EOSEUR,crypto,EOS-Euro,EUR,1,0.001,0.1
+gdax,EOSUSD,crypto,EOS-US Dollar,USD,1,0.001,0.1
+gdax,ETCBTC,crypto,Ethereum Classic-Bitcoin,BTC,1,0.000001,0.1
+gdax,ETCEUR,crypto,Ethereum Classic-Euro,EUR,1,0.001,0.1
+gdax,ETCGBP,crypto,Ethereum Classic-Pound Sterling,GBP,1,0.001,0.1
+gdax,ETCUSD,crypto,Ethereum Classic-US Dollar,USD,1,0.001,0.1
+gdax,ETHBTC,crypto,Ethereum-Bitcoin,BTC,1,0.00001,0.01
+gdax,ETHDAI,crypto,Ethereum-Dai,DAI,1,0.01,0.01
+gdax,ETHEUR,crypto,Ethereum-Euro,EUR,1,0.01,0.01
+gdax,ETHGBP,crypto,Ethereum-Pound Sterling,GBP,1,0.01,0.01
+gdax,ETHUSD,crypto,Ethereum-US Dollar,USD,1,0.01,0.01
+gdax,ETHUSDC,crypto,Ethereum-USDC,USDC,1,0.01,0.01
+gdax,GNTUSDC,crypto,Golem-USDC,USDC,1,0.000001,1.
+gdax,LIKETH,crypto,ChainLink-Ethereum,ETH,1,0.00000001,1.
+gdax,LIKUSD,crypto,ChainLink-US Dollar,USD,1,0.00001,1.
+gdax,LOMUSDC,crypto,Loom Network-USDC,USDC,1,0.000001,1.
+gdax,LTCBTC,crypto,Litecoin-Bitcoin,BTC,1,0.000001,0.1
+gdax,LTCEUR,crypto,Litecoin-Euro,EUR,1,0.01,0.1
+gdax,LTCGBP,crypto,Litecoin-Pound Sterling,GBP,1,0.01,0.1
+gdax,LTCUSD,crypto,Litecoin-US Dollar,USD,1,0.01,0.1
+gdax,MKRBTC,crypto,Maker-Bitcoin,BTC,1,0.00001,0.00000001
+gdax,MKRUSDC,crypto,Maker-USDC,USDC,1,0.00001,0.00000001
+gdax,MNAUSDC,crypto,Decentraland-USDC,USDC,1,0.000001,1.
+gdax,REPBTC,crypto,Augur-Bitcoin,BTC,1,0.000001,0.1
+gdax,REPEUR,crypto,Augur-Euro,EUR,1,0.00001,0.00000001
+gdax,REPUSD,crypto,Augur-US Dollar,USD,1,0.01,0.1
+gdax,XLMBTC,crypto,Stellar Lumens-Bitcoin,BTC,1,0.00000001,1.
+gdax,XLMEUR,crypto,Stellar Lumens-Euro,EUR,1,0.000001,1.
+gdax,XLMUSD,crypto,Stellar Lumens-US Dollar,USD,1,0.000001,1.
+gdax,XRPBTC,crypto,Ripple-Bitcoin,BTC,1,0.00000001,1.
+gdax,XRPEUR,crypto,Ripple-Euro,EUR,1,0.0001,1.
+gdax,XRPUSD,crypto,Ripple-US Dollar,USD,1,0.0001,1.
+gdax,XTZBTC,crypto,Tezos (Pre-Launch)-Bitcoin,BTC,1,0.00000001,1.
+gdax,XTZUSD,crypto,Tezos (Pre-Launch)-US Dollar,USD,1,0.0001,1.
+gdax,ZECBTC,crypto,Zcash-Bitcoin,BTC,1,0.000001,0.01
+gdax,ZECUSDC,crypto,Zcash-USDC,USDC,1,0.01,0.01
+gdax,ZILUSDC,crypto,Zilliqa-USDC,USDC,1,0.00001,0.00000001
+gdax,ZRXBTC,crypto,0x-Bitcoin,BTC,1,0.00000001,1.
+gdax,ZRXEUR,crypto,0x-Euro,EUR,1,0.000001,1.
+gdax,ZRXUSD,crypto,0x-US Dollar,USD,1,0.000001,1.
```

Comment:
Thank you for fixing the lot sizes, also nice to have the complete coin/token description 👍